js/scripts.js

```diff
@@ -0,0 +1,49 @@
+var mobileMenuExpand = document.querySelector('#mobilemenuexpand');
+var expanded = document.querySelector('#expanded');
+var sub = document.querySelector('#sub');
+var mobileMenu = document.querySelector('#mobilemenu');
+
+mobileMenu.addEventListener ("click", function(){
+	mobileMenuExpand.style.display = "flex"
+});
+
+var exitMobileMenu = document.querySelector('#exit');
```

Comment:
oops this was a remnant of an experiment that i have since fully deleted before seeing your comment -- i added an X icon to the mobile menu that i used to close it, but then realized the original designs dont have an exit button so i stopped trying to develop it.